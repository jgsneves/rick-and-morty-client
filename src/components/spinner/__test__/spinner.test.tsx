import { render } from '@testing-library/react';
import Spinner from '..';

describe('Spinner test suite', () => {
  it('should render primary color snapshot', () => {
    const { asFragment } = render(<Spinner color="primary" size={50} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render secondary color snapshot', () => {
    const { asFragment } = render(<Spinner color="secondary" size={50} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
