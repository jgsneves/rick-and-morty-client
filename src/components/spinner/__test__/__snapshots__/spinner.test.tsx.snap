// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner test suite should render primary color snapshot 1`] = `
<DocumentFragment>
  <svg
    class="sc-bczRLJ iBHIyA"
    color="primary"
    size="50"
    viewBox="0 0 50 50"
  >
    <circle
      class="path"
      cx="25"
      cy="25"
      fill="none"
      r="20"
      stroke-width="6"
    />
  </svg>
</DocumentFragment>
`;

exports[`Spinner test suite should render secondary color snapshot 1`] = `
<DocumentFragment>
  <svg
    class="sc-bczRLJ jccGrK"
    color="secondary"
    size="50"
    viewBox="0 0 50 50"
  >
    <circle
      class="path"
      cx="25"
      cy="25"
      fill="none"
      r="20"
      stroke-width="6"
    />
  </svg>
</DocumentFragment>
`;
